pkg/services/ngalert/models/notifications_test.go

```diff
@@ -201,6 +201,55 @@ func TestNotificationSettingsLabels(t *testing.T) {
 	}
 }
 
+func TestNotificationSettings_OldFingerprint(t *testing.T) {
```

Comment:
Copied from [here](https://github.com/grafana/grafana/blob/6f83a6b2fdfc6d3198c5c53be1480d08df5b13db/pkg/services/ngalert/models/notifications_test.go#L115) and slightly adapted to test `FingerprintOld`